pages/edit_group.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
```

Comment:
We don't need shebangs in any of our Python files as they're not executed from the command line. It would be good to remove them entirely so they stop reproducing via copy+paste.